jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/AbstractConnection.java

```diff
@@ -140,17 +131,14 @@ protected void failedCallback(final Callback callback, final Throwable x)
      */
     public void fillInterested()
     {
-        if (LOG.isDebugEnabled())
-            LOG.debug("fillInterested {}", this);
-        getEndPoint().fillInterested(_readCallback);
+        fillInterested(_readCallback);
```

Comment:
I'm growing a feeling that this method should be removed and be replaced by `fillInterested(Callback callback)`.